package.json

```diff
@@ -1,21 +1,23 @@
 {
-  "name": "cyf-ui",
-  "version": "0.1.0",
-  "private": true,
+  "name": "@codeyourfuture/cyf-ui",
+  "version": "4.2.0",
+  "module": "/dist/index.js",
+  "private": false,
   "dependencies": {
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-scripts": "3.0.1",
     "@storybook/addon-actions": "^5.1.10",
     "@storybook/addon-links": "^5.1.10",
     "@storybook/addons": "^5.1.10",
-    "@storybook/react": "^5.1.10"    
+    "@storybook/react": "^5.1.10",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "3.0.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
+    "babel:build": "babel src/components/index.js --out-dir dist",
```

Comment:
You need to change this line to: 
`  "babel:build": "babel src/components --out-dir dist",`
Because it looks like Babel doesn't transpile and include the imports, i.e.the components. So we need to transform the entire components folder and include it our package as right now only index.js is being included.